.github/workflows/_reusable-verify-run-status.yaml

```
@@ -0,0 +1,39 @@
+# Prefix with "~" to sort last in Actions list
+name: ~Templates - Verify Run Status
+
+on:
+  workflow_call:
+    inputs:
+      check_run_name:
+        description: Name of the check run to verify
+        required: true
+        type: string
+      workflow_name:
+        description: Name of the workflow to verify
+        required: true
+        type: string
+
+permissions:
+  checks: read
+  contents: read
+
+jobs:
+  check-run-status:
+    if: ${{ (github.event_name == 'workflow_run') || (github.event_name == 'check_suite' && github.event.check_suite.app.name == 'openapi-pipeline-app') || (github.event_name == 'check_run' && github.event.check_run.name == inputs.check_run_name) }}
```

```suggestion
    if: |
      (github.event_name == 'workflow_run') ||
      (github.event_name == 'check_suite' && github.event.check_suite.app.name == 'openapi-pipeline-app') ||
      (github.event_name == 'check_run' && github.event.check_run.name == inputs.check_run_name)
```

Can split conditional across lines, but must use this syntax instead of `${{ }}`